reports/report_builder.py

```diff
@@ -172,7 +183,13 @@ def create_journalist_export(self, model, wb):
                 else:
                     ws.write(row+y,col+x, getattr(obj, field.name))
             # Write link to end of row
-            ws.write_url(row+y, col+x+1, "%s/%s/%s/" % (self.edit_url, parent_model._meta.model_name, parent_id))
+            # ws.write_url(row+y, col+x+1, "%s/%s/%s/" % (self.edit_url, parent_model._meta.model_name, parent_id))
```

Comment:
Usually best not to leave lines of code commented out. Just remove them, they're always in git.